docs/standard/modernize-with-azure-and-containers/index.md

```diff
@@ -0,0 +1,194 @@
+
+There isn't a single, one-size-fits-all strategy for migrating applications to the cloud. The right migration strategy for you will depend on your organization's needs and priorities, and the kind of applications you are migrating. Not all applications warrant the investment of moving to a platform as a service ([PaaS](https://azure.microsoft.com/overview/what-is-paas/)) model or developing a [cloud-native](https://www.gartner.com/doc/3181919/architect-design-cloudnative-applications) application model. In many cases, you can take a phased or incremental approach to invest in moving your assets to the cloud, based on your business needs.
+
+For modern applications with the best long-term agility and value for the organization, you might benefit from investing in *cloud-optimized* and *cloud-native* application architectures. However, for applications that are existing or legacy assets, the key is to spend minimal time and money (no re-architecting or code changes) while moving them to the cloud, to realize significant benefits.
+
+Figure 1-1 shows the possible paths you can take when you move existing .NET applications to the cloud in incremental phases.
+
+ ![Modernization paths for existing .NET applications and services](./media/image1-1.png)
+
+> **Figure 1-1**. Modernization paths for existing .NET applications and services
+
+Each migration approach has different benefits and reasons for using it. You can choose a single approach when you migrate apps to the cloud, or choose certain components from multiple approaches. Individual applications aren't limited to a single approach or maturity state. For instance, a common hybrid approach would have certain on-premises components plus other components in the cloud.
+
+At the first two migration levels, you can just lift and shift your applications:
+
+**Level 1: Cloud Infrastructure-Ready**: In this migration approach, you simply rehost or move your current on-premises applications to an infrastructure as a service ([IaaS](https://azure.microsoft.com/overview/what-is-iaas/)) platform. Your apps have almost the same composition as before, but now you deploy them to VMs in the cloud.
+
+**Level 2: Cloud DevOps-Ready**: At this level, after an initial lift and shift, and still without re-architecting or altering code, you can gain even more benefits from running your app in the cloud. You improve the agility of your applications to ship faster by refining your enterprise development operations (DevOps) processes. You achieve this by using technologies like Windows Containers, which is based on Docker Engine. Containers remove the friction that's caused by application dependencies when you deploy in multiple stages. Containers also use additional cloud managed services related to data, monitoring, and continuous integration/continuous deployment (CI/CD) pipelines.
```

Comment:
technologies like Windows Containers, which is based... --> a technology like Windows Containers, which is based